README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
 If you are running Debian or Ubuntu, you can make sure that these are
 installed by running:
 
-    sudo apt-get install default-jre perl python wget rdiff-backup socat iptables
+`sudo apt-get install default-jre perl python wget rdiff-backup socat iptables`
```

Comment:
Everywhere else we talk about running a command, it has spaces before the command rather than using the back-tick marks around the command.  Since the back-ticks and spaces have the same affect (placing the code in a box), I would prefer to keep this with spaces to be more readable from the command line.  If there weren't enough spaces to get this in a box (it looks like it may have been short), there should be other examples in the README that have the correct number of spaces.